dev-utils/run-script.js

```diff
@@ -24,42 +24,38 @@
  */
 
 const { join } = require('path')
+const runAll = require('npm-run-all')
 const {
   runKarma,
   runSauceConnect,
   runE2eTests: runE2eTestsUtils,
   buildE2eBundles: buildE2eBundlesUtils
 } = require('./test-utils')
-const runAll = require('npm-run-all')
-
 const { startTestServers } = require('./test-servers')
-
-const {
-  getSauceConnectOptions,
-  getTestEnvironmentVariables
-} = require('./test-config')
+const { getTestEnvironmentVariables } = require('./test-config')
 const { generateNotice } = require('./dep-info')
 
 const PROJECT_DIR = join(__dirname, '../')
-
-const sauceConnectOpts = getSauceConnectOptions()
 const { sauceLabs } = getTestEnvironmentVariables()
 
-function runUnitTests(launchSauceConnect = 'false', directory) {
-  const karmaConfigFile = join(__dirname, '..', directory, './karma.conf.js')
-  if (launchSauceConnect === 'true' && sauceLabs) {
-    return runSauceConnect(sauceConnectOpts, () => runKarma(karmaConfigFile))
+function runUnitTests(packagePath, startSauceConnect = 'false') {
+  const karmaConfigFile = join(PROJECT_DIR, packagePath, 'karma.conf.js')
+  if (startSauceConnect === 'true') {
+    return launchSauceConnect(() => runKarma(karmaConfigFile))
   }
   runKarma(karmaConfigFile)
 }
 
-function launchSauceConnect() {
+/**
+ * Checks if the MODE is set to "saucelabs" and decides where to run the
+ * corresponding callback function
+ */
+function launchSauceConnect(callback = () => {}) {
   if (sauceLabs) {
-    return runSauceConnect(sauceConnectOpts, () => {
-      console.log('Launched SauceConnect!')
```

Comment:
I think we should keep this log, it's useful to know at which point sauce connect was launched.

Reply:
sauce connect already logs it when connected, so we were double logging the same stuff hence removed it . Check this example log

```
Starting sc with args: -u XXXXXXXX -k XXXXXXXX --no-ssl-bump-domains all --tunnel-identifier 752.3 --readyfile /tmp/sc-launcher-readyfile
Creating tunnel with Sauce Labs
Detected sc ready
Testing tunnel ready
Sauce connect ready
```